chore: Regenerate section API to accept multiple text files uploads

asymness opened the pull request:
Resolves https://github.com/Unstructured-IO/community/issues/11

- [x] Re-generate sections API with incremented version.
- [x] Update/add unit tests.

**Usage Example**:
```shell
pip install requests-toolbelt
```

```python
import requests
import json
from requests_toolbelt.multipart import decoder
from base64 import b64decode

files=[
  ('text_files',('pepg-S-1-1835597-000119312522106884.xbrl',open('sample-sec-docs/pepg-S-1-1835597-000119312522106884.xbrl','rb'),'application/octet-stream')),
  ('text_files',('rgld-10-K-85535-000155837021011343.xbrl',open('sample-sec-docs/rgld-10-K-85535-000155837021011343.xbrl','rb'),'application/octet-stream'))
]

response = requests.post("http://localhost:8000/sec-filings/v0.2.0/section", files=files, data=dict(section='RISK_FACTORS'), stream=True)
data = decoder.MultipartDecoder.from_response(response)
results = []
for part in data.parts:
    results.append(json.loads(b64decode(part.content)))

with open('multipart-response-parsed.json', 'w+') as f:
    json.dump(results, f, indent=4)
```

cragwolfe reviewed (commented):
Please update README curl commands to reflect change in parameter name.

Also, let's go ahead and bump the version and references in the files listed here: https://github.com/Unstructured-IO/pipeline-sec-filings/blob/b3faef4/Makefile#L204-L207

asymness commented:
@cragwolfe, updated the parameter in README commands and update the versions.
Also added an example of parsing the `multipart/mixed` response in the PR description.
